config/defaults.js

```diff
@@ -300,4 +300,13 @@
     * NOTE: When an empty array is specified, an empty object of responseHeaders will be returned
     */
     logResponseHeaders: 'all',
+
+    /**
+     * Retry argument for file download
+     * // TODO FIX
+     * Applies to FileDownloads and to bulk-client's attachment downloads
+     */
+    fileDownloadRetry: {
+        limit: 3,
```

Comment:
@mikix (who requested this change and is our most active user) uses 5 retries in his temp. workaround. Should we default to 5?

Also, the `TODO FIX` in comment is confusing. Is there anything left to fix here?

Reply:
Love that! And the TODO fix does make sense here – I need to update the comment above. Thank you for flagging.